e2e/test/scenarios/dashboard/dashboard-card-resizing.cy.spec.js

```diff
@@ -0,0 +1,171 @@
+import {
+  addOrUpdateDashboardCard,
+  editDashboard,
+  getDashboardCards,
+  restore,
+  saveDashboard,
+  visitDashboard,
+} from "e2e/support/helpers";
+import { SAMPLE_DB_ID } from "e2e/support/cypress_data";
+import { ORDERS, ORDERS_ID, PEOPLE } from "metabase-types/api/mocks/presets";
```

Comment:
Wrong import, there is another one with IDs for e2e tests

Reply:
Ah ok, I was going to ask about that, something seemed really off